build-system/tasks/presubmit-checks.js

```diff
@@ -307,7 +307,7 @@ const forbiddenTerms = {
       'extensions/amp-fx-collection/0.1/providers/fx-provider.js',
       'extensions/amp-list/0.1/amp-list.js',
       'extensions/amp-next-page/0.1/next-page-service.js',
-      'extensions/amp-next-page/0.2/service.js',
+      'extensions/amp-next-page/0.2/visibility-observer.js',
```

Comment:
I don't have all the context for this change; what is this whitelist used for?

Reply:
This just whitelists the dependency of the file on the `PositionObserver`